.env.test

```diff
@@ -14,4 +14,6 @@ MONGO_CONNECTION_STRING=mongodb://fakemongo.com
 MONGO_DATABASE_NAME="TEST_todo-db"
 # Credentials for docker mongodb instance
 MONGO_INITDB_ROOT_USERNAME=root
-MONGO_INITDB_ROOT_PASSWORD=root
\ No newline at end of file
+MONGO_INITDB_ROOT_PASSWORD=root
+
+LOGIN_DISABLED=True
```

Comment:
It would be nice to also update your `.env.template` file with the new values typically required:

```
GITHUB_OAUTH_CLIENT_ID=id
GITHUB_OAUTH_URL=https://github.com/login/oauth
GITHUB_OAUTH_CLIENT_SECRET=secret
HOMEPAGE_URL=http://localhost:5000
```